src/Resources/skeleton/web/app.php

```diff
@@ -21,6 +22,19 @@
 /** @var Composer\Autoload\ClassLoader */
 $loader = require __DIR__.'/../vendor/autoload.php';
 
+if (file_exists(__DIR__.'/../.env')) {
+    (new Dotenv())->load(__DIR__.'/../.env');
+}
```

Comment:
Is that needed in the `app.php`?

Reply:
Ah, https://github.com/contao/manager-bundle/pull/69, got it ;)

Reply:
Not really related to #69, but we need it if people want to configure production using .env files. I agree it's not Symfony best practice, but 90% of our people don't have real server variables.